stix_shifter_modules/splunk/stix_translation/json/from_stix_map.json

```diff
@@ -125,21 +125,22 @@
         "fields": {
             "src_ref.value": "src",
             "src_port": "src_port",
-            "dst_ref.value": "src",
+            "dst_ref.value": "dest",
             "dst_port": "dest_port",
             "protocols[*]": "protocol",
             "start": "earliest",
             "end": "latest"
         }
     },
     "process": {
-        "cim_type": "process",
+        "cim_type": "endpoint",
```

Comment:
I guess `process` cim type doesn't exist anymore?  
I'm not sure how relevant it is to use `"cim_type"` in the from_stix_map since we integrated all the mappings into one from_stix.  that's another problem to look at though.

Reply:
I do not know what `cim_type` is used for.